manifests/ldap.pp

```diff
@@ -34,9 +34,9 @@
   String              $sync_pw,
   String              $sync_hash,
   String              $base_dn    = simplib::ldap::domain_to_dn(),
-  String              $bind_dn    = "cn=hostAuth,ou=Hosts,${simp_options::ldap::base_dn}",
-  String              $sync_dn    = "cn=LDAPSync,ou=Hosts,${simp_options::ldap::base_dn}",
-  String              $root_dn    = "cn=LDAPAdmin,ou=People,${simp_options::ldap::base_dn}",
+  String              $bind_dn    = "cn=hostAuth,ou=Hosts,${base_dn}",
```

Comment:
This should have no impact on anything.

Reply:
@jeefberkey Is that good or bad? It just works around a bug in Puppet 5